osf/models/node.py

```diff
@@ -391,6 +395,24 @@ def parent_node(self):
                 return parent
         return None
 
+    @property
+    def storage_limit_status(self):
+        """ This should indicate if a node is at or over a certain storage threshold indicating a status. If nodes have
+        a custom limit this should indicate that."""
+
+        limits = dict([(item.name, item.value) for item in settings.StorageLimits])
+        if self.custom_storage_usage_limit_public:
+            limits['OVER_PUBLIC'] = self.custom_storage_usage_limit_public
+            warning_limit = self.custom_storage_usage_limit_public * settings.STORAGE_WARNING_THRESHOLD
+            limits['APPROACHING_PUBLIC'] = warning_limit
+        if self.custom_storage_usage_limit_private:
+            limits['OVER_PRIVATE'] = self.custom_storage_usage_limit_private
+            warning_limit = self.custom_storage_usage_limit_private * settings.STORAGE_WARNING_THRESHOLD
+            limits['APPROACHING_PRIVATE'] = warning_limit
+
+        limits = enum.IntEnum('StorageLimitsWithCustomValues', [(key, value) for key, value in limits.items()])
+        return max(limit for limit in limits if limit.value <= self.storage_usage)
```

Comment:
I believe `storage_usage` is going to be in bytes, so you'll probably need some translation here, otherwise we'll just always be over public limit for the most part.

Reply:
We should also clarify if we're using 10^9 or 2^30 as our conversion for GB to Bytes